CONTRIBUTING.md

```diff
@@ -101,6 +101,27 @@ How to edit OpenAPI file:
     - `Delete`(single record), 
     - `Create`(single record), 
     - `CreateInBulk`(multiple record)
+- If a 0-level model has dedicated sub-models (contains only `oneOf`), then those model's names should follow the pattern:
+  `{Resource}{Action}{Differentiator}{Request|Response}{Body|Query}`
+  where the  `Differentiator` describes the child model, e.g.:
+    - `Base [PublicationsCreateBaseResponseBody]` (common part of other child models)
+    - `Vouchers [PublicationsCreateVouchersResponseBody]`
+    - `Voucher [PublicationsCreateVoucherResponseBody]`
+```json
```

Comment:
Dodaj jeszcze proszę komentarz, że title:
Should be the same like schema name but in `Title Case`.
Oraz, że description:
Should be applied to scheme {Request|Response}{Body|Query} schema for **{METHOD} `{path}`.
If multiple paths add next one after `and`